k6ext/panic.go

```diff
@@ -36,3 +44,20 @@ func Panic(ctx context.Context, format string, a ...interface{}) {
 	_ = p.Release()
 	_ = p.Kill()
 }
+
+type userFriendlyError struct{ err error }
+
+func (e *userFriendlyError) Unwrap() error { return e.err }
+
+func (e *userFriendlyError) Error() string {
+	switch {
+	default:
+		return e.err.Error()
+	case e.err == nil:
+		return ""
+	case errors.Is(e.err, context.DeadlineExceeded):
+		return strings.ReplaceAll(e.err.Error(), context.DeadlineExceeded.Error(), "timed out")
```

Comment:
As you mentioned in the commit message, it would be useful if we could log the deadline time as well. But if it's a lot of work, don't bother. This is an improvement already.